src/components/newsletter/newsletter-sign-up.tsx

```diff
@@ -0,0 +1,264 @@
+    label {
+      width: 1px;
+    }
+  }
+`;
+
+type SignUpStatus = {
+  success: boolean;
+  msg: string;
+};
+
+const NewsletterSignUp = () => {
+  const [email, setEmail] = useState(undefined);
+  const [emailError, setEmailError] = useState(false);
+  const [name, setName] = useState(undefined);
+  const [nameError, setNameError] = useState(false);
+  const [mcResponse, setMcResponse] = useState<SignUpStatus | undefined>(
+    undefined
+  );
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+
+    setMcResponse(undefined);
+
+    if (!name) setNameError(true);
+    if (!email) setEmailError(true);
+    if (!name || !email) return;
+
+    const response = await addToMailchimp(email, { FNAME: name });
+
+    handleResult(response.result, response.msg);
+  };
+
+  const handleResult = (result: string, message: string) => {
+    const success = result === "success";
+    if (!success) {
+      const msg = message.includes("already subscribed")
+        ? "This email has already been subscribed."
+        : "Something went wrong, please try again later.";
+      setMcResponse({ success, msg });
+    } else {
+      setMcResponse({ success, msg: "Thank you for subscribing!" });
+    }
+  };
+
+  return (
+    <>
+      <section css={newsletterSignUpStyles}>
+        <header>
+          <h2>Join our mission</h2>
+          <p>
+            We’ve got bold ambitions and need impact focused leaders to join us
+            on our journey. Sign up to our newsletter to be the first to hear
+            about our events and stay up to date on our future plans.
+          </p>
+        </header>
+
+        <article>
+          <form onSubmit={handleSubmit}>
+            <fieldset>
+              <section>
+                <label htmlFor="fullNameInput">Full name</label>
+                <input
+                  type="text"
+                  name="fullName"
+                  id="fullNameInput"
+                  className={nameError ? "error" : ""}
+                  onChange={e => {
+                    setNameError(false);
+                    setName(e.target.value);
+                  }}
+                />
+                <p className="input-error">Please enter your name.</p>
+              </section>
+              <section>
+                <label htmlFor="emailInput">Email address</label>
+                <input
+                  type="email"
+                  name="email"
+                  id="emailInput"
+                  className={emailError ? "error" : ""}
+                  onChange={e => {
+                    setEmailError(false);
+                    setEmail(e.target.value);
+                  }}
+                />
+                <p className="input-error">Please enter your email.</p>
+              </section>
+              <section className="submitSection">
+                <label htmlFor="submit">&nbsp;</label>
+                <input type="submit" value="Submit" id="submit" />
```

Comment:
the designs say `Submit`, it's true, but in the paragraph below it says `Subscribe` – can we change the button to say `Subscribe`? @ramsayalbazzaz @carinapope13